gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -562,6 +572,9 @@ func (vm *VMKeeper) QueryFile(ctx sdk.Context, filepath string) (res string, err
 		return memFile.Body, nil
 	} else {
 		memPkg := store.GetMemPackage(dirpath)
+		if memPkg == nil {
+			return "", fmt.Errorf("package %q is not available", dirpath) // TODO: XSS protection
```

Comment:
Leftover todo?

Reply:
Actually, copy-paste from the above.

I can delete them both with the rationale that if XSS protection is to be added, this is a stupid place to add it anyway. wdyt?

Reply:
I think it's fine to leave it, just wanted to check 👍